locationsharinglib/locationsharinglib.py

```diff
@@ -274,7 +274,26 @@ def _get_data(self):
         self._logger.debug(response.text)
         try:
             output = json.loads(response.text.split("'", 1)[1])
-            people = [Person(info) for info in output[0]]
+            people = []
+            if output[0] is not None:
```

Comment:
Output can never be none, so checking for it does not make sense.

Reply:
Output can't be none, but the element at index 0 _can_ be none if no one is sharing their location with the account.

Reply:
Good point.I will check a non shared account tomorrow and get back to you. Thanks!